### ISSUE TITLE
Help me crop my screenshot window

### ISSUE THREAD

**Issue by andrewisom** [nbclark/hqcheat#17]
I am on a 2017 macbook air taking the screenshot from an iPhone 8 Plus. Can someone help me figure out the correct values of the cropping variables? I keep getting errors.

**Comment by betatesta144** [comment]
I need help as well haha all it detects is swipe left for comments

**Comment by andrewisom** [comment]
Yes same here!

**Comment by betatesta144** [comment]
Lets try to fix this together. I believe if we just mess around with the resolutions we can get it to display. Right now it is going to far down.

**Comment by andrewisom** [comment]
Same here. But when i adjust it to get the full screen, it gives me an error

**Comment by betatesta144** [comment]
If you look at the video that comes in one of the folders it seems that he keeps the window in the left and not full screen so try doing that. Right now im editing the index.js to see if I can get it to work

**Comment by andrewisom** [comment]
Okay im working on it too. lmk if you find anything that works

**Comment by betatesta144** [comment]
Ok same here.

**Comment by betatesta144** [comment]
Right now I am trying to figure what resolutions go where in the file. There are 2 places to put them in. :/

**Comment by betatesta144** [comment]
Ok I found something. So if you comment out the command that deletes the cropped photo you can see the photo. Right now the photo screenshots the bottom right.

**Comment by andrewisom** [comment]
I saw that too. And when I take a screenshot of the whole question it gives me an error.

**Comment by betatesta144** [comment]
yeah. I need to get it to go up to the middle and right then the square would perfectly capture. It seems he got the capture zone correct but not the area

**Comment by andrewisom** [comment]
We seemed to have gotten the same area so maybe it doesn't matter which phone you have

**Comment by betatesta144** [comment]
Yeah it just matters how big the quicktime window is. Also it seems that the directions are messed up or something because when I edit like top it doesnt effect the top it effects like the sides

**Comment by andrewisom** [comment]
yes i noticed that too! ughhh what is the optimal window

**Comment by betatesta144** [comment]
haha im not sure but I edited top to about 700 and got progress because that told us that basically higher the number further down the screen it goes

**Comment by andrewisom** [comment]
hmmmm

**Comment by betatesta144** [comment]
I don't believe you can use negative numbers as well.

**Comment by betatesta144** [comment]
Do you know how to open png.2 files? I have to use the info tab to look at them

**Comment by andrewisom** [comment]
Rename them so that they are just .png files and they will display correctly

**Comment by betatesta144** [comment]
Ok thanks. Also one more thing what are the units that are for the area? Are they px or?

**Comment by andrewisom** [comment]
i think px but not totally sure

**Comment by betatesta144** [comment]
Ok. Ive yet figured out which directions edit what

**Comment by betatesta144** [comment]
https://stackoverflow.com/questions/22589322/what-does-top-left-right-and-bottom-mean-in-android-rect-object Found this. It may be useful

along with this
https://github.com/chenglou/png-crop/blob/master/README.md

**Comment by betatesta144** [comment]
Sorry not to bother you again but im just really eagered to get this done but do you know how to put the screenshot area up instead of only showing the comments or maybe if we cropped the live video on quicktime

**Comment by andrewisom** [comment]
did you figure it out?

**Comment by betatesta144** [comment]
Nope. Im going to hop on it again in a bit. How about you?

**Comment by betatesta144** [comment]
But today I was thinking that we can move the area and it is just naturaly in the corner and we are going to have to find a different way around it

**Comment by D-Buz** [comment]
EVERYONE PLEASE WRITE YOUR MODEL COMPUTER AND PHONE THAT YOU ARE TRYING TO USE, AND I WILL MAKE A REPOSITORY WITH ALL THE FORMATTING OPTIONS YOU COULD NEED.

**Comment by andrewisom** [comment]
@D-Buz 2017 MacBook Air 13 inch, iPhone 8 Plus. Thank you!!!!

**Comment by betatesta144** [comment]
@D-Buz 13.3 in 2012 Macbook Pro, iPhone X. Thank you so much.

**Comment by betatesta144** [comment]
@D-Buz This is awesome that you are doing this. When do you suppose the repos be up?

**Comment by AlexPerez01** [comment]
@D-Buz any update on this repo?

**Comment by darksignup1** [comment]
2017 MacbookPro, iPhone 7+, please help me out with the resolution!

**Comment by darksignup1** [comment]
@D-Buz

**Comment by ram1434** [comment]
Linux, lenovo p70 pls help me
